skore-ui/src/components/CrossValidationReport.vue

```diff
@@ -0,0 +1,73 @@
+<script lang="ts">
+export interface ScalarResult {
+  name: string;
+  value: number;
+  fold?: number;
+}
+
+export interface TabularResult {
+  name: string;
+  columns: any[];
+  data: any[][];
+  favorability: "higher-is-better" | "lower-is-better";
+}
+
+export interface PrimaryResults {
+  scalarResults: ScalarResult[];
+  tabularResults: TabularResult[];
+}
+</script>
+
+<script setup lang="ts">
+import CrossValidationReportResults from "@/components/CrossValidationReportResults.vue";
+import TabPanel from "@/components/TabPanel.vue";
+import TabPanelContent from "@/components/TabPanelContent.vue";
+
+const fake: PrimaryResults = {
+  scalarResults: [
+    { name: "toto", value: 4.32 },
+    { name: "tata", value: 4.32 },
+    { name: "titi", value: 4.32, fold: 1 },
+    { name: "tutu", value: 4.32 },
+  ],
+  tabularResults: [
+    {
+      name: "a",
+      columns: ["Accuracy", "Precision", "Recall", "F1 Score"],
+      data: [
+        [0.8, 0.4, 0.5, 0.6],
+        [0.8, 0.4, 0.5, 0.6],
+        [0.8, 0.4, 0.5, 0.6],
+        [0.8, 0.4, 0.5, 0.6],
+      ],
+      favorability: "higher-is-better",
```

Comment:
Every metric can have a different "favorability", did you group the "higher-is-better" metrics together on purpose?

Reply:
Seems that this is not part of the "MVP", so I removed it. 
To be discussed in the "backend" PR imho.